doc/review.qbk

```diff
@@ -0,0 +1,148 @@
+[/
+    Copyright (c) 2013-2017 Vinnie Falco (vinnie dot falco at gmail dot com)
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+]
+
+[section:review For Reviewers]
+
+To set realistic expectations and prevent a litany of duplicate review
+statements, these notes address the most common questions and comments
+about Beast and other HTTP libraries that have gone through formal review.
+
+[variablelist
+[[
+    "Beast requires too much user code to do anything!"
+][
+    It is not the intention of the library to provide turn-key
+    solutions for specific HTTP or WebSocket use-cases.
+    Instead, it is a sensible protocol layering on top of
+    Boost.Asio which retains the Boost.Asio memory
+    management style and asynchronous model. 
+]]
+[[
+    "Beast does not offer an HTTP server?"
+][
+    The scope of the library is to provide a message container,
+    and serialization and deserialization algorithms, with an
+    interface permitting users to achieve all the behaviors
+    necessary to implement high performance, conforming servers.
+    However, management of listening sockets, io_service threads,
+    individual connections, and HTTP semantics such as pipelining,
+    redirection, Expect: 100-continue, cookies, request routing,
```

Comment:
You probably want "and" after "cookies" and also ditch the comma after "request routing"